bin/ciserve.sh

```diff
@@ -15,4 +15,4 @@ if [ -z "$TMPDIR" ]; then
 fi
 
 # Start a local copy of the site to test against
-bundle exec jekyll serve --destination $TMPDIR/_pa11y_site
+bundle exec jekyll serve --destination $TMPDIR/_pa11y_site  >> _site/_serve_output.txt 2>&1
```

Comment:
`>>` redirects stderr to the file. Then `2>&1` redirects stderr to stdout. Did you mean to redirect stdout to the file? If so, use a single `>`.